README.md

```diff
@@ -1,3 +1,11 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+To successfully connect to the two databases locally follow the instructions:
+
+1. Clone the repo to your machine.
+2. Create two .env files: '.env.test' and '.env.development'.
+3. Within each file add the line PGDATABASE=database_name_here, database_name_here should be replaced with the actual name of the database.
```

Comment:
You can just specify the names of your databases here. Please replace ```database_name_here``` with the names of the databases to add to each file 🙂